robottelo/utils/virtwho.py

```diff
@@ -541,3 +568,48 @@ def get_configure_command_option(deploy_type, args, org=DEFAULT_ORG):
         return f"hammer -u {username} -p {password} virt-who-config deploy --id {args['id']} --organization-title '{args['organization-title']}' "
     elif deploy_type == 'name':
         return f"hammer -u {username} -p {password} virt-who-config deploy --name {args['name']} --organization '{org}' "
+
+
+def vw_fake_conf_create(
+    owner,
+    rhsm_hostname,
+    rhsm_username,
+    rhsm_encrypted_password,
+    fake_conf_file,
+    json_file,
+    is_hypervisor=True,
+):
+    conf_name = fake_conf_file.split("/")[-1].split(".")[0]
+    file = f'{fake_conf_file}\n'
+    title = f'[{conf_name}]\n'
+    type = 'type=fake\n'
+    json = f'file={json_file}\n'
+    is_hypervisor = f'is_hypervisor={is_hypervisor}\n'
+    owner = f'owner={owner}\n'
+    env = 'env = Library\n'
+    rhsm_hostname = f'rhsm_hostname={rhsm_hostname}\n'
+    rhsm_username = f'rhsm_username={rhsm_username}\n'
+    rhsm_encrypted_password = f'rhsm_encrypted_password={rhsm_encrypted_password}\n'
+    rhsm_prefix = 'rhsm_prefix=/rhsm\n'
+    rhsm_port = 'rhsm_port=443\n'
+    cmd = ('cat <<EOF > {}' '{}' '{}' '{}' '{}' '{}' '{}' '{}' '{}' '{}' '{}' '{}' 'EOF').format(
```

Comment:
How about removing `\n` from variables and:
```
f'cat <<EOF > {file}\n{title}\n{type}\n...'
```

Reply:
> How about removing `\n` from variables and:
> 
> ```
> f'cat <<EOF > {file}\n{title}\n{type}\n...'
> ```

Thank you very much for your good suggestion ,have already updated